tdrs-backend/manifest.yml

```diff
@@ -6,3 +6,15 @@ applications:
   disk_quota: 2G
   docker:
     image: ((docker-backend))
+  env:
+    AV_SCAN_URL: http://((cf-space))-clamav-rest.apps.internal:9000/scan
+- name: clamav-rest
+  instances: 1
+  memory: 1.5G
+  disk_quota: 2G
+  docker:
+    image: rafttech/clamav-rest:0.103.2
+  env:
+    MAX_FILE_SIZE: 200M
+  routes:
+  - route: ((cf-space))-clamav-rest.apps.internal
```

Comment:
Routes must be *globally unique* (across all customers) so we will append the space (`tanf-dev`, `tanf-staging`, tanf-prod`) to the beginning of the route and run only 1 instance of clamav-rest per space.